app/services/avatar_deleter.rb

```diff
@@ -0,0 +1,19 @@
+class AvatarDeleter < ApplicationService
+  def initialize(user, params)
+    @user = user
+    @params = params
+  end
+
+  def call
+     if @user.image.attached? && @params[:deleteImage] == "1"
+      @user.image.purge
+    end
```

Comment:
you could have just one line like: 
```suggestion
     @user.image.purge if @user.image.attached? && (@params[:deleteImage] == "1")
```